csrc/scheduler/vectorize_helper.cpp

```diff
@@ -840,6 +840,14 @@ int64_t getVectorizationFactor(
         max_vec_size,
         SchedulerRuntimeInfo::max_alignment_size_in_byte / dtype_size);
 
+    // Handle misaligned inputs or outputs. Usually tensors are fully aligned at
```

Comment:
Isn't the code below also for checking alignment?

Reply:
Oh you're right. Not sure how I missed that. I'll strip out these changes to only the part that keeps the actual tensors in `getMaybeHeuristicsFor`.